docs/rule/kotlin_library.soy

```diff
@@ -60,6 +60,18 @@ of the compiled class files and resources.
   {/param}
 {/call}
 
+{call buck.arg}
+  {param name: 'annotation_processing_tool' /}
+  {param default : 'kapt' /}
+  {param desc}
+  Specifies the tool to use for annotation processing. Possible values: "kapt" or "javac".
+  "kapt" allows to run Java annotation processors against Kotlin sources while backporting
```

Comment:
s/"kapt" allows to run Java annotation/"kapt" allows to running Java annotation/